i18n/fr/docusaurus-plugin-content-docs/version-22.04/monitoring/discovery/hosts-discovery.md

```diff
@@ -62,30 +62,35 @@ ressemblerait le résultat de la découverte :
 
 ### Étape 5 : Définir les politiques d'analyse et de mise à jour
 
-- **Analyse manuelle** : Une fois la tâche de découverte exécutée, dans la liste des hôtes découverts, l'utilisateur choisira manuellement quels 
-    hôtes ajouter à la page **Configuration > Hôtes > Hôtes** (voir [Analyser le résultat d'une tâche de découverte](#analyser-le-résultat-dune-tâche-de-découverte)).
+- **Analyse manuelle** : Une fois la tâche de découverte exécutée, dans la liste des hôtes découverts,
+l'utilisateur choisira manuellement quels hôtes ajouter à la page **Configuration > Hôtes > Hôtes**
+(voir [Analyser le résultat d'une tâche de découverte](#analyser-le-résultat-dune-tâche-de-découverte)).
 
-- **Analyse automatique** : L'analyse traitera le résultat automatiquement selon l'option sélectionnée (au moins une option doit être cochée) :
+- **Analyse automatique** : L'analyse traitera le résultat automatiquement selon l'option sélectionnée (au moins une option parmi les 3 premières doit être cochée) :
```

Comment:
You don't need to check an option from the first 3
If you only check the last one, it will update existing hosts